modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -3335,7 +3323,7 @@ private CompletableFuture<IgniteBiTuple<RowId, Collection<Lock>>> takeLocksForUp
      * @return Future completes with tuple {@link RowId} and collection of {@link Lock}.
      */
     private CompletableFuture<IgniteBiTuple<RowId, Collection<Lock>>> takeLocksForInsert(BinaryRow binaryRow, RowId rowId, UUID txId) {
-        return lockManager.acquire(txId, new LockKey(tableId()), LockMode.IX) // IX lock on table
+        return lockManager.acquire(txId, new LockKey(replicationGroupId), LockMode.IX)
```

Comment:
Why did we change tableId to replicationGroupId in all places where the loca key is used with single parameters, but not do that for the two parametrazed constructors?

Reply:
Good point, fixed.